src/background/service/APIService.js

```diff
@@ -755,7 +755,9 @@ class APIService {
                 const tw = oasis.staking.allowWrapper()
                 params.method = TRANSACTION_TYPE.StakingAllow
                 params.toAddress = oasis.staking.addressToBech32(await oasis.staking.addressFromRuntimeID(oasis.misc.fromHex(params.runtimeId)))
-                let result = await this.submitTxBody(params, tw,true,(data)=>this.depositToParatimeAccount(params,resolve,reject,data)).catch(err=>err)
+                let result = await this.submitTxBody(params, tw, true, (data) => {
```

Comment:
no need to use curly braces here

Reply:
My preference is to have a curly for multiline

Reply:
also nice to use braces for statements (even single statements imo) and no braces for expressions